tests/test_fairworkflow.py

```diff
@@ -48,6 +48,16 @@ def test_iterator(self):
         for i, step in enumerate(workflow_steps):
             assert step == right_order_steps[i]
 
+    def test_validate_inputs_outputs(self):
+        #
+        self.step1.outputs = ['var1']
+        self.step2.inputs = ['var1']
+        self.workflow.validate()
+        self.step1.inputs = ['var1']
+        self.step2.outputs = ['var1']
```

Comment:
So, just to get things straight in my head: in the above the output called ```var1``` is actually, in principle, a URI that uniquely identifies that output? Or can it also be just a string literal? URI would definitely be preferable, since then we can refer to it in other triples.

I expect these URIs will end up looking like ```www.purl.org/stepuri#outputvarname``` or something along those lines?

Reply:
You are correct! It is a URI. I will make it clear in the documentation for inputs and outputs.